src/components/Templates/AccessSettings/AccessSettings.scss

```diff
@@ -44,6 +44,10 @@
   gap: $spacing--md;
 }
 
+.access-settings__main .select-option {
```

Comment:
as the css class you're selecting is in another file (`SelectOption.tsx`), why not just change `SelectOption.scss` respectively?